content/en/docs/concepts/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Concepts
-description: The collection of requirements and scenarios that define v2 of the Notary project
+description: The collection of requirements and scenarios that define The Notary Project
```

Comment:
Please note that `Notary Project` is the new brand name as described in https://github.com/notaryproject/.github/issues/35